model_info/srwiki.articletopic.md

```diff
@@ -0,0 +1,961 @@
+		Culture.Media.Radio                          0.935
+		Culture.Media.Software                       0.988
+		Culture.Media.Television                     0.986
+		Culture.Media.Video games                    0.989
+		Culture.Performing arts                      0.968
+		Culture.Philosophy and religion              0.952
+		Culture.Sports                               0.981
+		Culture.Visual arts.Architecture             0.978
+		Culture.Visual arts.Comics and Anime         0.985
+		Culture.Visual arts.Fashion                  0.962
+		Culture.Visual arts.Visual arts*             0.964
+		Geography.Geographical                       0.972
+		Geography.Regions.Africa.Africa*             0.973
+		Geography.Regions.Africa.Central Africa      0.979
+		Geography.Regions.Africa.Eastern Africa      0.956
+		Geography.Regions.Africa.Northern Africa     0.977
+		Geography.Regions.Africa.Southern Africa     0.97
+		Geography.Regions.Africa.Western Africa      0.908
+		Geography.Regions.Americas.Central America   0.96
+		Geography.Regions.Americas.North America     0.967
+		Geography.Regions.Americas.South America     0.974
+		Geography.Regions.Asia.Asia*                 0.967
+		Geography.Regions.Asia.Central Asia          0.971
+		Geography.Regions.Asia.East Asia             0.98
+		Geography.Regions.Asia.North Asia            0.976
+		Geography.Regions.Asia.South Asia            0.974
+		Geography.Regions.Asia.Southeast Asia        0.969
+		Geography.Regions.Asia.West Asia             0.974
+		Geography.Regions.Europe.Eastern Europe      0.974
+		Geography.Regions.Europe.Europe*             0.959
+		Geography.Regions.Europe.Northern Europe     0.97
+		Geography.Regions.Europe.Southern Europe     0.976
+		Geography.Regions.Europe.Western Europe      0.974
+		Geography.Regions.Oceania                    0.975
+		History and Society.Business and economics   0.958
+		History and Society.Education                0.961
+		History and Society.History                  0.959
+		History and Society.Military and warfare     0.972
+		History and Society.Politics and government  0.955
+		History and Society.Society                  0.905
+		History and Society.Transportation           0.985
+		STEM.Biology                                 0.981
+		STEM.Chemistry                               0.987
+		STEM.Computing                               0.987
+		STEM.Earth and environment                   0.973
+		STEM.Engineering                             0.981
+		STEM.Libraries & Information                 0.961
+		STEM.Mathematics                             0.97
+		STEM.Medicine & Health                       0.979
+		STEM.Physics                                 0.983
+		STEM.STEM*                                   0.979
+		STEM.Space                                   0.995
+		STEM.Technology                              0.976
+		-------------------------------------------  -----
+	pr_auc (micro=0.775, macro=0.601):
```

Comment:
A few topics are low utility (e.g. "Society" and "Math") but this is mostly good.  I wonder if we could ask a local what is going on with the math content.  Maybe there's some other languages that seep into the math articles.

Reply:
What does low utility mean?

Reply:
Essentially, I think the topics will be less useful in these wikis when the fitness for those topics is low.  @geohci has a new method for getting math labels.  I don't think we have tried that yet.  It might have a big impact here.

Reply:
Hmm...I can't remember what you're referring to here (there is a readjustment of the Society-related wikiprojects that I haven't submitted but can soon if that's desired). Only thing I could think of is e.g., adding a feature similar to pronouns that counts <math> tags in the wikitext but I'd actually be a bit cautious of adding that feature because while it would help with a big chunk of mathematics articles, i think it would probably mean that mathematics was only ever predicted when there are <math> symbols and would largely kill the utility of this topic for new articles.

Reply:
I thought you made some improvements to your process that allowed us to use the Math Rating template or something like that.

Reply:
We might just need to update the dataset we download from figshare and use that -- if I'm remembering correctly.

Reply:
Ahh yes -- you jogged my memory :) Morten had made an excellent comment about maths rating being a commonly used template for WikiProject Mathematics. In the groundtruth used by this repo (https://github.com/wikimedia/drafttopic/blob/master/Makefile#L27), it looks like there are only about 120 articles with the WikiProject Mathematics label (more end up in the Mathematics topics via Statistics, Cryptography, etc.). In the more recent versions of the groundtruth data though (`labeled_enwiki...` here: https://figshare.com/articles/Wikipedia_Articles_and_Associated_WikiProject_Templates/10248344/4), I used the page assessments table and that thankfully automatically handled the maths rating issue and there are about 20000 articles under WikiProject Mathematics. some of these are probably already included via statistics etc. but presumably making the change to the new groundtruth would improve things. that said, the page assessments table uses descriptors that are not the same as the wikiproject name so you have to be careful when joining in the data to the taxonomy. specifically, i applied this python function to both the page assessments names and the wikiproject names in the taxonomy to get them to align:
<code>
def norm_wp_name_en(wp):
    """Normalize WikiProject names in English"""
    ns_local = 'wikipedia'
    wp_prefix = 'wikiproject'
    return re.sub("\s\s+", " ", wp.lower().replace(ns_local + ":", "").replace(wp_prefix, "").strip())
</code>
you'd probably have to do something similar here: https://github.com/wikimedia/drafttopic/blob/e35ff59574593276a6f2652202fc8ff197a7e9a0/drafttopic/utilities/taxo_label.py#L72